app/models/qa/authorities/local/find_works.rb

```diff
@@ -0,0 +1,23 @@
+module Qa::Authorities
+  class Local::FindWorks < ActiveFedora::Base
+    attr_reader :q
+
+    def initialize(q)
+      @q = q
```

Comment:
Without this init, I get an error : `ArgumentError (wrong number of arguments (given 1, expected 0))` even after extending `Qa::Authorities::Base` as requested above.